mlbench_core/utils/pytorch/distributed.py

```diff
@@ -25,6 +25,8 @@ def aggregate_gradients(model, world_size, average_models=False):
 def global_average(sum, count):
     def helper(array):
         array = torch.Tensor(array)
+        if dist.get_backend() == "nccl":
```

Comment:
Why do you hard-code "nccl" here instead of using `dist.Backend.NCCL` as in partition.py? I think we should use `dist.Backend.NCCL` if possible.

Reply:
Yes it would be better to use `dist.Backend.NCCL` when possible